wbo/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: wbo
+      APP_PORT: 80
+
+  server:
+    container_name: wbo
+    volumes:
+      - ${APP_DATA_DIR}/wbo-boards:/opt/app/server-data
+    image: impranshu/wbo:1@sha256:1458870c58a319638bcf7d474b76fe4b2eafb1020a97444680345a60d0e7896c
```

Comment:
Shouldn't we put the official docker image here ?

https://hub.docker.com/r/lovasoa/wbo

Reply:
Yeah. Absolutely. At that time I didn't find the official docker image so I made a docker image and went ahead. Will be modifying the code and opening the PR soon. Thanks

Reply:
I have updated the link and verified that it's working fine. Thanks

Reply:
I still see `impranshu/wbo` instead of `lovasoa/wbo`